code/game/machinery/alarm.dm

```diff
@@ -576,22 +576,22 @@
 	data["temperature_c"]=round(environment.temperature - T0C, 0.1)
 
 	var/percentages[0]
-	percentages["oxygen"]=oxygen_percent
-	percentages["nitrogen"]=nitrogen_percent
+	percentages[GAS_OXYGEN]=oxygen_percent
+	percentages[GAS_OXYGEN]=nitrogen_percent
 	percentages["co2"]=co2_percent
```

Comment:
I'm not sure if you'd also need to update something somewhere else if you changed this to `GAS_CARBON`, but if not you should